teams/team-geordie/dev/yelb/templates/deployment.yaml

```diff
@@ -0,0 +1,157 @@
+apiVersion: v1
+kind: Service
+metadata:
+  namespace: geordie
+  name: redis-server
+  labels:
+    app: redis-server
+    tier: cache
+spec:
+  type: ClusterIP
+  ports:
+    - port: 6379
+  selector:
+    app: redis-server
+    tier: cache
+---
+apiVersion: v1
+kind: Service
+metadata:
+  namespace: geordie
+  name: yelb-db
+  labels:
+    app: yelb-db
+    tier: backenddb
+spec:
+  type: ClusterIP
+  ports:
+    - port: 5432
+  selector:
+    app: yelb-db
+    tier: backenddb
+---
+apiVersion: v1
+kind: Service
+metadata:
+  namespace: geordie
+  name: yelb-appserver
+  labels:
+    app: yelb-appserver
+    tier: middletier
+spec:
+  type: ClusterIP
+  ports:
+    - port: 4567
+  selector:
+    app: yelb-appserver
+    tier: middletier
+---
+apiVersion: v1
+kind: Service
+metadata:
+  namespace: geordie
+  name: yelb-ui
+  labels:
+    app: yelb-ui
+    tier: frontend
+spec:
+  type: LoadBalancer
```

Comment:
we will have to do something about proliferation of services of type loadbalancer. We can defer for now, but we need a task to refactor the deployments to use ingress controller , so that we get one lb for all of the services per cluster.

Reply:
`ho11y` does not have Ingress by design and `yelb` supports ingress as an option but they do have `LoadBalancer` support so we can keep it as it is now as these are sample apps. We can handle this later.

Reply:
let's create a follow-up in the blueprints backlog.  Could be a good "first task" for a new contributor trying to learn the system.